lib/currency_cloud/fake_resource.rb

```diff
@@ -0,0 +1,11 @@
+module CurrencyCloud
+  class FakeResource
+    include CurrencyCloud::Resource
+
+    def set_something_with_a_long_method()
+       wrongly_indented = 'not sure about this var'
```

Comment:
Useless assignment to variable - `wrongly_indented`.<br>Use 2 (not 3) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.